### PR TITLE
sidecar-shim: README: fix misinterpretation

### PR DESCRIPTION
**Pull request by victortoso**
More than once, users thought that /usr/bin/onDefineDomain could be a folder where they install their hook binary. Fix it.

### What this PR does
Before this PR:

Users considered that their hook binary could be installed in a folder named `/usr/bin/onDefineDomain`

After this PR:

It should be clear that the binary full path is `/usr/bin/onDefineDomain`

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:
none

The following alternatives were considered:
none

Links to places where the discussion took place: 
none

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

File: cmd/sidecars/README.md [hunk 1]
```diff
@@ -40,7 +40,8 @@ requires a `--version` parameter (e.g: v1alpha2)
 ## Example
 
 Using the current [smbios sidecar](../example-hook-sidecar/) as example. The `smbios.go` is compiled
-and installed in `/usr/bin/onDefineDomain` in a container that uses `sidecar-shim-image` as base.
+and the binary is named `onDefineDomain` and installed in `/usr/bin` in a container that uses
+`sidecar-shim-image` as base which has `/sidecar-shim` as the entrypoint.
```
Comment (brianmcarey) [inline cmd/sidecars/README.md:44]:
```suggestion
as a binary named `onDefineDomain` and installed under `/usr/bin` in a container that uses
`sidecar-shim-image` as base with an entrypoint of `/sidecar-shim`.
```

**Review by brianmcarey — COMMENTED** [review]
Thanks for this @victortoso ! I have left a slightly reworded suggestion mainly just to avoid the multiple "ands".
Let me know what you think.

**Comment by victortoso** [comment]
> Let me know what you think.

Thanks @brianmcarey, looks great. I squashed the change to keep this as a single commit.

**Review by brianmcarey — COMMENTED** [review]
/lgtm

**Comment by alicefr** [comment]
Much better, thanks @victortoso!
/approve

**Pull request merged**